server/.env_template

```diff
@@ -0,0 +1,7 @@
+MQTT_HOST=
+MQTT_USERNAME=
+MQTT_PASSWORD=
+MQTT_PORT=
+MQTT_WEBSOCKET_PORT=
+MQTT_TOPIC=
+MQTT_MESSAGE=
```

Comment:
Should we remove this? Consider the content of messages is pretty dynamic

Reply:
Hmm will do. I'll add a refactoring ticket though and do it in another PR. Want to get this merged in so this functionality can be in `main`